article/_posts/2020-09-04-infogan.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: review
+title: "InfoGAN: Interpretable Representation Learning by Information Maximizing Generative Adversarial Nets"
+tags: deep-learning GAN 
+author: "Nathan Painchaud"
+cite:
+    authors: "Xi Chen, Yan Duan, Rein Houthooft, John Schulman, Ilya Sutskever, Pieter Abbeel"
+    title:   "InfoGAN: Interpretable Representation Learning by Information Maximizing Generative Adversarial Nets"
+    venue:   "Advances in Neural Information Processing Systems, 2016"
+pdf: "https://arxiv.org/pdf/1606.03657.pdf"
+---
+
+
+![](/article/images/infogan/sc.png)
+(Image taken from [HERE](https://towardsdatascience.com/infogan-generative-adversarial-networks-part-iii-380c0c6712cd))
+
+# Highlights
+InfoGAN is an GAN which can learn a **disantangled** latent space distribution in an unsupervised manner.  
+
+# Introduction
+A basic GAN is illustrated with the graph in the previous figure but without the red boxes.  The main idea of InfoGAN is to add two little things to the basic GAN (the red boxes!).
+
+First and foremost, InfoGAN use an extra latent vector (called code): $$c=(c_0, c_1, ..., c_n)$$ that is appended to the original latent input vector $$z$$.  Doing this turns the original generator $$G(z)$$ into $$G(z,c)$$.   
```

Comment:
"(called code)": Maybe emphasize _code_ here in some way (e.g. italic, bold, etc.) to make it stand out?